lib/mpl_toolkits/axisartist/axes_divider.py

```diff
@@ -1,2 +1,2 @@
-from mpl_toolkits.axes_grid1.axes_divider import (  # noqa
+from ..axes_grid1.axes_divider import (  # noqa: F401 # pylint: disable=W0611
```

Comment:
Why change these to relative imports? I thought the explicit versions were preferred?